README.md

```diff
@@ -1,17 +1,17 @@
 # Leaderboard
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> This is a project created during my Microverse career where I needed to build a website that creates a Leaderboard for scores added by the user
+> This is a project created during my Microverse career where I built a website that creates a Leaderboard for game scores added by the user
 
-![screenshot](./screenshot.png)
+![screenshot](./images/screenshot.png)
 
 ## Built With
 
 - HTML, CSS and Javascript
```

Comment:
- [ ] I think you have missed mentioning the command to run the project. i.e. `npm start`. Please mention this in the README file.